rotkehlchen/externalapis/etherscan.py

```diff
@@ -118,6 +121,9 @@ def __init__(
             SupportedBlockchain.SCROLL,
         ) else 'api-'
         self.base_url = base_url
+        self.full_url: str = ''
```

Comment:
why keep both base and full url? Is this required?

Reply:
I've reworked this a bit. Now its called `legacy_api_domain` and `api_url` - which should hopefully be a bit clearer, and also removes the `prefix_url` attribute. Now `api_url` holds the url currently in use, and `legacy_api_domain` holds the etherscan domain used when creating the url for the old api version.